Taxes/Operation/01_ManageDecisionMatrix.md

```diff
@@ -0,0 +1,256 @@
+
+  > [Info] There are many regular expressions testing tools available free of charge in the Internet, such as [Regex101](https://regex101.com/).
+
+  For detailed information about this function, see [Define tax exceptions on ZIP code level](#define-tax-exceptions-on-ZIP-code-level).
+
+6. Click the *Net value of goods smaller than* drop-down list and select the applicable currency. If necessary, specify an amount using the keyboard or the increase and decrease arrows on the right.
+
+7. Click the *Tax zone* drop-down list and select the applicable tax zone.
+
+8. Click the [SAVE] button.  
+A pop-up window confirms that the decision matrix has been saved. The new entry has been saved.
+
+  ![Decision matrix saved](../../Assets/Screenshots/Taxes/Settings/DecisionMatrix/DecisionMatrixSaved.png "[Decision matrix saved]")
+
+
+## Edit an entry
+
+Edit an entry to modify any of the configured parameters or to change its position in the decision matrix.
+
+#### Prerequisites  
+
+An entry in the decision matrix is created, see [Create an entry](#create-an-entry).
+
+#### Procedures
+
+*Taxes > Settings > Tab DECISION MATRIX*
+
+![Decision matrix](../../Assets/Screenshots/Taxes/Settings/DecisionMatrix/DecisionMatrix.png "[Decision matrix]")
+
+1. Click the drop-down list to be edited.  
+The selected field becomes active and the corresponding drop-down list is displayed.
+
+2. Select a different option from the drop-down list.  
+The newly selected option is displayed in the field.
+
+3. If an option from a drop-down list is already selected, click the selected option again to deselect it.  
+The option **Any** (wildcard) is displayed in the field.
+
+4. Click the *Ship-to postcode* drop-down list on the left to select a different option, if necessary, and/or the field on the right to modify the range or the regular expression. Use the keyboard to enter a different value.
+
+5. Click the *Net value of goods smaller than* drop-down list on the left to select a different currency, if necessary, and/or the field on the right to modify the net value amount entered. You can use the keyboard or the increase and decrease arrows.
+
+6. Click the *Tax zone* drop-down list to select a different applicable tax zone.
+
+7. To modify the entry position in the matrix, click the ![Drag](../../Assets/Icons/Points03.png "[Drag]") (Drag) button to the left of the entry line, move the entry to a different position with the left mouse button pressed and drop it in the desired position by releasing the mouse button.   
+
+  > [Info] It is important to remember that the correct order of the decision-making criteria is crucial, as the system goes through the criteria in the order they are listed.
+
+8. Click the [SAVE] button.  
+A pop-up window confirms that the decision matrix has been saved. The edited entry has been saved.
+
+  ![Decision matrix saved](../../Assets/Screenshots/Taxes/Settings/DecisionMatrix/DecisionMatrixSaved.png "[Decision matrix saved]")
```

Comment:
Das ist etwas durcheinander. Erst gibt es ein paar Schritte, in denen du generell von irgendwelchen Drop-down-Listen sprichst und dann gehst du fast alle listen nochmal einzeln durch. Entweder du hältst es allgemeingültig, oder du gehst auf alle listen bzw. felder in einem gesonderten Schritt ein. 

Ganz raus lassen und in eine eigene Prozedur stecken würde ich den schritt 7. (vielleicht ## edit the decision matrix ### edit an entry  ### change the order of entries?!)